qiskit/visualization/text.py

```diff
@@ -779,33 +772,36 @@ def wire_names(self, with_initial_state=False):
             initial_qubit_value = ""
             initial_clbit_value = ""
 
+        self._bits_regs_map = get_bits_regs_map(
+            self._circuit, (self.qubits + self.clbits), self.cregbundle
+        )
         # quantum register
```

Comment:
Is this comment still relevant?

Reply:
Removed in https://github.com/Qiskit/qiskit-terra/pull/7319/commits/55e62189a965a13f2fee8bdb9cdfb1f9cb88d3fd.